rectools/models/sasrec.py

```diff
@@ -406,8 +545,32 @@ def process_dataset_train(self, dataset: Dataset) -> Dataset:
         user_id_map = IdMap.from_values(interactions[Columns.User].values)
         item_id_map = IdMap.from_values(self.item_extra_tokens)
         item_id_map = item_id_map.add_ids(interactions[Columns.Item])
+
+        # get item features
+        item_features = None
+        if dataset.item_features is not None:
+            item_features = dataset.item_features
+            if not isinstance(item_features, SparseFeatures):
+                raise ValueError("`item_features` in `dataset` must be `SparseFeatures` instance.")
+
+            internal_ids = dataset.item_id_map.convert_to_internal(
+                item_id_map.get_external_sorted_by_internal()[self.n_item_extra_tokens :]
+            )
+            sorted_item_features = item_features.take(internal_ids)
+
+            dtype = sorted_item_features.values.dtype
+            n_features = sorted_item_features.values.shape[1]
+            pad_item_features = sparse.csr_matrix((self.n_item_extra_tokens, n_features), dtype=dtype)
+
+            extra_tokens_feature_values: sparse.scr_matrix = sparse.vstack(
```

Comment:
```suggestion
            full_feature_values: sparse.scr_matrix = sparse.vstack(
```
Naming must be logical. Always)